### PR TITLE
Makes shadowling possible as a mulligan antag

### PR DESCRIPTION
**Pull request by ghost**
So yeah, if a wizard fucks up bad, it might turn into shadowlings!

:cl: Thefastfoodguy
tweak: Makes shadowling possible as a mulligan antag
/:cl:

### CODE REVIEW

**Comment by PopNotes** [comment]
+1 dis

**Comment by ghost** [comment]
shadowlings require pop
mulligan tends to occur when pop is low due to death

**Comment by ghost** [comment]
@McBawbaggings i thought mulligan sometimes happened during like, unrobust wizards / blob

**Comment by ghost** [comment]
it always happens on antag death on mulligan modes provided there is enough crew

even if an antag kills half of the crew then it could feasibly mulligan

shadowlings are not a good mode to mulligan into similar to rev, gang and cult

**Comment by MoreRobustThanYou** [comment]
also this will require https://github.com/tgstation/tgstation/pull/29846 to be merged and mirrored

**Comment by ghost** [comment]
this should be merge ready, i fixed it to where mulligan antags now check for minimum playercount

**Comment by KingPhilipIII** [comment]
I thought slings were removed when Hippie rebased to TG code.

**Comment by ghost** [comment]
We readded them

File: code/game/gamemodes/game_mode.dm [hunk 1]
```diff
@@ -120,7 +120,7 @@
 	var/list/datum/game_mode/runnable_modes = config.get_runnable_midround_modes(living_crew.len)
 	var/list/datum/game_mode/usable_modes = list()
 	for(var/datum/game_mode/G in runnable_modes)
-		if(G.reroll_friendly)
+		if(G.reroll_friendly && living_crew >= G.required_players)
```
Comment (ghost) [inline code/game/gamemodes/game_mode.dm]: This is not needed, runnable_modes already is checked on required_players

**Comment by ghost** [comment]
Ok, fixed.

**Comment by ghost** [comment]
merge this plz

**Comment by ghost** [comment]
guessing this won't be merged. *shrug*

**Comment by ghost** [comment]
conversion gamemodes that require alive players are not a good gamemode to mulligan in to

**Comment by ghost** [comment]
@McBawbaggings by default it checks to see if there's enough alive players

**Comment by Carbonhell** [comment]
Mcbaw didnt +1 this,and i dont like this a lot aswell. Waiting for other headmin's input.

**Comment by ghost** [comment]
Ok.

**Pull request closed without merging**